rcgen/tests/generic.rs

```diff
@@ -358,6 +358,29 @@ mod test_parse_other_name_alt_name {
 	}
 }
 
+#[cfg(feature = "x509-parser")]
+mod test_csr_extension_request {
+	use rcgen::{CertificateParams, KeyPair, KeyUsagePurpose};
+	use x509_parser::prelude::{FromDer, ParsedExtension};
+
+	#[test]
+	fn dont_write_sans_extension_if_no_sans_are_present() {
+		let mut params = CertificateParams::default();
+		params.key_usages.push(KeyUsagePurpose::DigitalSignature);
+		let key_pair = KeyPair::generate().unwrap();
+		let csr = params.serialize_request(&key_pair).unwrap();
+		let (_, parsed_csr) =
+			x509_parser::certification_request::X509CertificationRequest::from_der(csr.der())
```

Comment:
Please import `X509CertificationRequest` directly instead of using the long qualified path (same for other tests added here).

Reply:
Done!